### DISCUSSION TITLE
请问这个是什么原因0.0

### DISCUSSION THREAD

**Discussion by ForsakenUD** [ByteLegend/ByteLegend#92] [Q&A]
![image](https://user-images.githubusercontent.com/57027904/140527714-cee8f6ad-81c2-441b-8f93-3e9fdb56acab.png)

**Comment by blindpirate** [comment]
建议使用IDE和构建工具。如果一定要用命令行操作，请至少保证对`javac`/`java`和classpath有基本的了解。

**Reply by ForsakenUD** [reply]
> 建议使用IDE和构建工具。如果一定要用命令行操作，请至少保证对`javac`/`java`和classpath有基本的了解。

好的，解决了，谢谢！！
